packages/multi-parser/package.json

```diff
@@ -0,0 +1,68 @@
+{
+  "name": "@asyncapi/multi-parser",
+  "version": "2.0.0",
+  "description": "This tool allows to parse AsyncAPI documents and produce a desired interface based on a given Parser-API version",
+  "private":false,
+  "bugs": {
+    "url": "https://github.com/asyncapi/parser-js/issues"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/asyncapi/parser-js.git"
+  },
+  "publishConfig": {
+    "access": "public"
+  },
+  "license": "Apache-2.0",
+  "homepage": "https://github.com/asyncapi/parser-js",
+  "sideEffects": false,
+  "main": "cjs/index.js",
+  "module": "esm/index.js",
+  "types": "esm/index.d.ts",
+  "files": [
+    "/esm",
+    "/cjs",
+    "LICENSE",
+    "README.md"
+  ],
+  "scripts": {
+    "build": "npm run build:esm && npm run build:cjs",
+    "build:esm": "tsc",
+    "build:cjs": "tsc --project ./tsconfig.cjs.json",
+    "test": "npm run test:unit",
+    "test:unit": "cross-env CI=true jest --coverage",
+    "lint": "eslint --max-warnings 0 --config ../../.eslintrc  --ignore-path ../../.eslintignore .",
+    "lint:fix": "eslint --max-warnings 0 --config .../../eslintrc  --ignore-path ../../.eslintignore . --fix",
+    "generate:readme:toc": "markdown-toc -i \"../../README.md\"",
+    "generate:assets": "npm run generate:readme:toc",
+    "bump:version": "npm --no-git-tag-version --allow-same-version version $VERSION",
+    "prepublishOnly": "npm run generate:assets"
+  },
+  "dependencies": {
+    "@asyncapi/avro-schema-parser": "^3.0.3",
+    "@asyncapi/openapi-schema-parser": "^3.0.4",
+    "@asyncapi/protobuf-schema-parser": "^3.0.0",
+    "@asyncapi/raml-dt-schema-parser": "^4.0.4",
+    "parserapiv1": "npm:@asyncapi/parser@^2.1.0",
+    "parserapiv2": "npm:@asyncapi/parser@3.0.0-next-major-spec.8",
+    "@asyncapi/parser": "file:../parser"
```

Comment:
Is there any reason for changing the dependency name? If there isnt, please do not modify the original code from https://github.com/smoya/multi-parser-js/blob/19750c914386d4417c9e17b13584634132805e27/package.json#L47

Reply:
Previously, we used parserv3, but now that the repositories are merged and multiparser is using the latest version, it makes sense to refer to it by the correct name. If anyone needs to know the version, they can simply check the version of parser.

Reply:
The reason the version was explicit is exactly because we wanted it to be explicit for each version

Reply:
ping @ayushnau

Reply:
ok fixed this.